metadata/metadata/controllers/submission/tests.py

```diff
@@ -99,16 +99,23 @@ def test_create_submission_with_db_down(self, mock_events, url_for):
     @mock.patch('metadata.controllers.submission.url_for')
     @mock.patch('metadata.controllers.submission.ev')
     def test_create_submission_with_invalid_event(self, mock_events, url_for):
-        """An internal server error is raised on an invalid event."""
+        """A Bad Request error is raised on an invalid event."""
         url_for.return_value = '/foo/'
-        mock_events.save.side_effect = InvalidEvent
+
+        def raise_invalid_event(*events, **kwargs):
```

Comment:
These tests were not actually working, but because unhandled exceptions were getting swallowed in the controllers I didn't notice. The controllers no longer swallow unhandled exceptions. And I rewrote these side-effects to actually raise InvalidEvent.